addon/styles/_frost-sidebar.scss

```diff
@@ -1,62 +1,52 @@
-.frost-sidebar .container {
-  width: 100%;
-  height: 100%;
-}
-
 .frost-sidebar {
   position: absolute;
-  left: 0;
+  left: 0px;
+
+  .liquid-container > .liquid-child {
+    width: auto;
+    height: auto;
+  }
+
+  .open-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
```

Comment:
We still need: display: flex; otherwise the tab moves down as shown here:
![sidebar-opencontainer-scss](https://cloud.githubusercontent.com/assets/18577537/17111061/eb9ffe46-526d-11e6-96ac-fd9b77ed46d5.png)